warehouse/static/sass/blocks/_project-description.scss

```diff
@@ -78,11 +78,14 @@
     font-weight: $bold-font-weight;
   }
 
-  em,
   i {
     font-style: italic
   }
 
+  em {
+    font-style: normal
```

Comment:
`em` is used in places which are not covered by the *.scss:
```bash
$ find -type f -iname *.html -print0 | xargs -0 grep '<em>'

./warehouse/admin/templates/admin/blacklist/confirm.html:                    <em>({{ role.role_name }})</em>
./warehouse/admin/templates/admin/emails/detail.html:      <li><em>{{ key }}</em>: {{ render_item(value) }}</li>
./warehouse/templates/accounts/profile.html:            <em>Last released on {{ project.releases[0].created|format_date() }}</em>
./warehouse/templates/includes/packaging/project-data.html:    <em>GitHub statistics:</em>
./warehouse/templates/manage/projects.html:            <em>Last released on {{ release.created|format_date() }}</em>
./warehouse/templates/manage/projects.html:            <em>Created on {{ project.created|format_date() }}</em>
./warehouse/templates/pages/help.html:          We use a number of terms to describe software available on PyPI, like <em>project</em>, <em>release</em>, <em>file</em>, and <em>package</em>. Sometimes those terms are confusing because they're used to describe different things in other contexts. Here's how we use them on PyPI:
./warehouse/templates/pages/help.html:          A <em>project</em> on PyPI is the name of a collection of releases and files, and information about them. Projects on PyPI are made and shared by other members of the Python community so that you can use them.
./warehouse/templates/pages/help.html:          A <em>release</em> on PyPI is a specific version of a project. For example, the <a href="https://pypi.org/project/requests/">requests</a> project has many releases, like <em>requests 2.10</em> and <em>requests 1.2.1</em>. A release consists of one or more <em>files</em>.
./warehouse/templates/pages/help.html:          A <em>file</em>, also known as a <em>package</em>, on PyPI is something that you can download and install. Because of different hardware, operating systems, and file formats, a release may have several files (packages), like an archive containing source code or a binary <a href="https://pypi.org/project/wheel/#files">wheel</a>.
./warehouse/templates/pages/help.html:          We welcome <em>constructive</em> feedback and bug reports via our <a href="https://github.com/pypa/warehouse/issues" target="_blank" rel="noopener">issue tracker</a>. Before writing a new issue, first check that a similar issue does not already exist.
./warehouse/templates/pages/help.html:        <em>Financial</em>: We would deeply appreciate <a href="https://donate.pypi.org/">your donations to fund development and maintenance</a>.
./warehouse/templates/pages/help.html:          <em>Development</em>: Warehouse is open source, and we would love to see some new faces working on the project. You <strong>do not</strong> need to be an experienced open-source developer to make a contribution – in fact, we'd love to help you make your first open source pull request!
./warehouse/templates/pages/help.html:          <em>Stay updated</em>: You can also follow the ongoing development of the project on the <a href="https://mail.python.org/mailman/listinfo/distutils-sig">distutils-sig mailing list</a> and the <a href="https://groups.google.com/forum/#!forum/pypa-dev" target="_blank" rel="noopener">PyPA Dev message group</a>.
./warehouse/templates/search/results.html:                {% if term %}for "<em>{{ term }}</em>"{% endif %}
./warehouse/templates/search/results.html:                  There were no results for '<em>{{ term }}</em>'
./warehouse/templates/search/results.html:                    There were no results for '<em>{{ filters }}</em>' filter
./warehouse/templates/search/results.html:                    There were no results for '<em>{{ filters }}</em>' filters
```
 where the default behaviour is used, as mentioned [here](https://www.w3schools.com/tags/tag_em.asp):
```css
em {
    font-style: italic;
}
```
and I think that in order to remove all italic styles, I would better remove/replace all the `em` tags, than changing the *.scss styles.
@di : what do you think?

Reply:
I think we should make it bold instead, like I suggested on the other line, but I'll defer to @nlhkabu here.